### DISCUSSION TITLE
Add required field to categories

### DISCUSSION THREAD

**Discussion by matthewcarleton** [ExpressionEngine/ExpressionEngine#2865] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I want to ensure that a category is chosen for an entry but I can't do that with native functionality. There is an addon that allows this but that addon doesn't allow users to add new categories from the publish page and I need to retain that functionality. 

**Describe the solution you'd like**
Make categories optionally required. There doesn't seem to be any reason to do this, it kind of feels like a bug.

**Comment by shbchk** [comment]
Any news on this? It would be super helpful!

**Comment by shbchk** [comment]
Found! https://github.com/ExpressionEngine/ExpressionEngine/pull/3555 It's coming in 7.4.0
